setup.py

```diff
@@ -54,7 +54,9 @@ def __init__(self, attrs=None):
     },
     package_data={
         'h-py':
-        ['out/*.dylib' if platform.system() == 'Darwin' else 'out/*.so.*']
+        ['out/*.dylib' if platform.system() == 'Darwin' else (
+            'out/*.dll' if platform.system() == 'Windows' else
```

Comment:
Now that I'm looking at this as a reviewer, I wonder if this should be `'out\\*.dll'` here to match how `libh3_path` is defined in `h3/h3.py` above?

Reply:
Updated, but I standardized on forward slashes in `h3/h3.py` instead.

Reply:
Does that work? I had the backslashes before because it wasn't concatenating the path correctly in Appveyor.

Reply:
Appears to be working in Appveyor: https://ci.appveyor.com/project/isaacbrodsky/h3-py/builds/23110297